packages/js/components/src/woo-product-field-item/woo-product-field-item.tsx

```diff
@@ -24,28 +24,35 @@ type WooProductFieldSlotProps = {
 
 export const WooProductFieldItem: React.FC< WooProductFieldItemProps > & {
 	Slot: React.FC< Slot.Props & WooProductFieldSlotProps >;
-} = ( { children, order = 20, section, id } ) => {
+} = ( { children, sections, id } ) => {
 	const { registerFill, getFillHelpers } = useSlotContext();
 
 	registerFill( id );
 
 	return (
-		<Fill name={ `woocommerce_product_field_${ section }` }>
-			{ ( fillProps: Fill.Props ) => {
-				return createOrderedChildren<
-					Fill.Props & SlotContextHelpersType,
-					{ _id: string }
-				>(
-					children,
-					order,
-					{
-						...fillProps,
-						...getFillHelpers(),
-					},
-					{ _id: id }
-				);
-			} }
-		</Fill>
+		<>
+			{ sections.map( ( { name: sectionName, order: sectionOrder } ) => (
+				<Fill
+					name={ `woocommerce_product_field_${ sectionName }` }
+					key={ sectionName }
+				>
+					{ ( fillProps: Fill.Props ) => {
+						return createOrderedChildren<
+							Fill.Props & SlotContextHelpersType,
+							{ _id: string }
+						>(
+							children,
+							sectionOrder || 20,
+							{
+								...fillProps,
+								...getFillHelpers(),
```

Comment:
I think we should expose the `sectionName` here as well.

Reply:
Done in e6543756c253907b96119baedec6401cd0448076